spec/javascripts/support/jasmine_helper.rb

```diff
@@ -62,8 +62,19 @@ def call(env)
     Jasmine::Runners::ChromeHeadless.new(formatter, jasmine_server_url, config)
   end
 
+  # use google-chrome-beta if available, otherwise use the default
+  if system('which google-chrome-beta &>/dev/null')
+    config.chrome_binary = 'google-chrome-beta'
+  end
+
+  # Determine the chrome binary and which version, so we can choose the correct headless mode
+  chrome_binary = Jasmine::Runners::ChromeHeadless.new(nil, nil, config).chrome_binary
+  chrome_version = `"#{chrome_binary}" --version`.strip
+  puts "Chrome binary: #{chrome_version}"
+  headless_mode = Gem::Version.new(chrome_version.split(" ").last) >= Gem::Version.new("110.0.5481.30") ? "new" : nil
```

Comment:
wow, that version number... .30 is the one that did it... not .29

Reply:
In case we need the exact version references in the future:

last good before it broke with beta `109.0.5414.74-1`: https://github.com/ManageIQ/manageiq-ui-classic/actions/runs/3897839430/jobs/6655930328#step:4:99
first bad after it broke with beta `110.0.5481.30-1` https://github.com/ManageIQ/manageiq-ui-classic/actions/runs/3907107044/jobs/6675972345#step:4:99